bpf/Makefile

```diff
@@ -46,10 +46,10 @@ LXC_OPTIONS = \
 	 -DSKIP_DEBUG \
 	 -DHAVE_LPM_MAP_TYPE \
 	 -DHAVE_LRU_MAP_TYPE \
-	 -DENABLE_IPV4 \
-	 -DENABLE_IPV6 \
-	 "-DENABLE_IPV6 -DENABLE_IPV4" \
-	 "-DENABLE_IPV4 -DENABLE_NAT46"
+	 -DDISABLE_IPV4 \
+	 -DDISABLE_IPV6 \
+	 "-DDISABLE_IPV6 -DDISABLE_IPV4" \
+	 "-DDISABLE_IPV4 -DENABLE_NAT46"
```

Comment:
NAT46 also needs this treatment, it used to be unconditionally enabled and now it's not.